README.md

```diff
@@ -67,7 +68,28 @@ jQuery is the swiss army knive for DOM and event handling and much more. While j
 ### How?
 
 * Use a lightweight alternative like [Dominus](https://github.com/bevacqua/dominus#readme), [Shoestring](https://github.com/filamentgroup/shoestring#readme) or [Zepto](https://github.com/madrobby/zepto#readme).
-* [Create a jQuery custom build](#create-a-custom-build) to only include the features you need.
+* [Create a jQuery custom build](#create-a-jquery-custom-build) to only include the features you need.
+
+[↑ back to Table of Contents](#table-of-contents)
+
+
+## Create a jQuery custom build
+
+### Why?
+
+Special builds can be created that exclude subsets of jQuery functionality. This allows for smaller custom builds when the builder is certain that those parts of jQuery are not being used. 
+
+### How?
+
+After cloning jQuery from the [public repository](https://github.com/jquery/), run:
+
+	npm install
+
+An then: 
+
+	grunt custom:-css,-deprecated,-effects,-event/alias,-offset
```

Comment:
Are these the parts that can be committed as a result o this guide? If so, great! If not, can we change that? It would also be nice to note the savings (kb / %) of this custom build.

I suppose this also requires `grunt`(`-cli`) to be installed globally. But let's leave that up to jQuery to explain :)